cms/templates/cms/toolbar/dragbar.html

```diff
@@ -25,6 +25,7 @@
 
     <div class="cms-dragbar-title">
         {{ placeholder.get_label }}
+        {% if placeholder.is_inherit %}<small>{% trans "Inherit Placeholder" %}</small>{% endif %}
```

Comment:
I would like to keep consistency with the other placeholder types, as https://github.com/divio/django-cms/issues/5037 suggests we should just display an icon instead of text.